docs/redoc-vendor-extensions.md

```diff
@@ -243,3 +243,40 @@ PayPalPayment:
 
 In the example above the names of definitions (`PayPalPayment`) are named differently than
 names in the payload (`paypal`) which is not supported by default `discriminator`.
+
+### Sort endpoints vendor extensions
```

Comment:
`x-sort-order` should be placed under `Operation Object vendor extensions` not to it's own section